docker/bootstrap/Dockerfile.mysql57

```diff
@@ -16,8 +16,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
         echo percona-server-server-5.7 percona-server-server/root_password password 'unused'; \
         echo percona-server-server-5.7 percona-server-server/root_password_again password 'unused'; \
     } | debconf-set-selections && \
-    percona-release enable-only tools \
-    apt-get update -y && \
+    apt-get update -y && apt-get install -y percona-release && \
+    percona-release enable-only tools && apt-get update -y && \
```

Comment:
Fix required to install percona.

Reply:
v20 does not support 5.7 any more, does it? Can't we delete this and the percona57 version?